src/components/helpers.js

```diff
@@ -0,0 +1,20 @@
+import classNames from 'classnames';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../constants';
+
+/**
+ * webex Component Classes defines class names for components
+ *
+ * @param {object} props Data passed to the component
+ * @param {string} props.classBaseName Component class base name
+ * @param {string} props.userClassName Custom CSS class to apply
+ * @param {string} props.otherClasses Other optional classes
+ * @returns {object} Classes
```

Comment:
isn't it a string?

Reply:
classBaseName and userClassName is string, but otherClasses and the returned classes are objects

Reply:
yes it is